docker/Readme.md

```diff
@@ -37,7 +37,7 @@ We have also developed a tool on top of docker that you could be interested in:
 
 
 ### Other Dockerfiles related with Robocomp
-This other Dockerfiles, dockercompose files and related images in docker hub could be of your interest if you are working with these projects.
+These other Dockerfiles, docker-compose files,and related images in the docker hub could be of your interest if you are working with these projects.
```

Comment:
```suggestion
These other Dockerfiles, docker-compose files, and related images in the docker hub could be of your interest if you are working with these projects.
```

Reply:
I don't remember to have changed anything to the "if" word, and it doesn't look like it's changing. I don't know why github is showing this as a change.